.travis.yml

```diff
@@ -0,0 +1,38 @@
+sudo: required
+
+services:
+- docker
+
+env:
+  global:
+  - secure: aaMCAyV9sd9Q1ZHq3yGAmGp3G87tA2cAgGrT7XucDnkjVviD+1ui4EsVm+0czMpsI8GCn3S69PDZPhWQTyP7Jt06LpH/op4Ccin2R+QbQ40bENP/1SrRaSvJUbE9ZkGIhE+8n0VmIafl21SfQi9chdcKwtF+Ow/zAtIJvo2/6dPE29n9JnKbFEAayeKj++TljS8+GJywoCF2/QaC7z76e8EUn0DwUup76RNfWaHL2jbw49+//hrt/LlGuGXL+B1+fy1r1esE5hSpsr7ln2WojNH1CfhyERiJxVPVaH/aS6gGVW+2VIX4pqL21KeA49ufV9/zuPmP4Bw1A/gQX8nGgZbsvjdP1CQ+AGlGDUZr8h9Dg2B2ybTY63mYzM1gZ3z3pGavMRlX40yNnjvVCFbTZnP40MDrpFPuUnyp+tHam+XqygwSWe70PC/MSvox/jXaFLYqs9YwkQvymZ7lgWA7mhQDeR7BaNcZRqpP74ESoibvbNfzSZINrIoX36NsoQOU4h7jT8iB7WBpI4oNzXJ8wVzOc3uQ4IbAVcWJU04SIWvHKSV/84fNO0ImchIQmTaKXhgLu6fQSz8NbhkOc3r/cJo2uP7U0p0/pdawTxwBc7CfNHCrMgJbCDiLR4xquqYzF/Yk7xfjF3KL5l9n66+NN3yg6SQafijSsEzNlUSngQo=
+  - secure: t3xZt9jM0WPgAcepXEuEW6W8kGai7vPAdcYPBclRyXFsoCsKLAloJfFuFr83hcb+M8GoB7OeNgszkMNJUyL83C5T8eybqN1R26RfgKgOWOMNS9lYWgfxLAKK56gHRtG8yr8mqM59/ZcJ0U+9JZxLpZYGwgHB0RJGMkYpO4tQ0s0znpRBGyLFfHkg3AJZu/Mq8LHFu7B02E9PGb9XSq1bAC8mZ2QVsIf0e6ObAcE2Deb9D1vmv6lozNpWzxf056GEWFgwmxC5DZNCKMHyQ5i9XIu/8lrJro9ipprZZIQrE1onNaSR06x8dInllfMIf6/3rBZU515Ekl4q5ssRGBQSw7ZD0w3/0FfPllhQUruYrD+ERk7CaqTq1Yy4khWy0PAD8TNOrb5zvSfYK2n2yq/d79QRwmCgN/B5aw/4tumxR3Y7RuIbcJKC+yDsI3rjZhjIFPK10GEw4hI5M8U6CRmqcqoiQdfxbPY4BeZ7m7X2VecHTng+H2RSZ+B7mo8kiT3R1Cyuq8IXM4ZwWeiyq9QQ1e8qoYYhGSZd+azTs4ymvVTiVPq0cvESJ0U3XH0TAZd3FaNrTPrJAjDo0xjVwLI6a0sdbQlKlv+0mhHEN+xGfCJxpkwcMPcqXM7sbWgNj9B87P3c4B/nMa/Ejvptbagst10MLruPebPVMn1Kn3oTCXg=
+  matrix:
+  - LATEST=0 SHOPWARE_VERSION=5.5.0RC1 PHP_VERSION=7.2
+  - LATEST=1 SHOPWARE_VERSION=5.4.6 PHP_VERSION=7.2
+  - LATEST=0 SHOPWARE_VERSION=5.4.6 PHP_VERSION=7.1
+  - LATEST=0 SHOPWARE_VERSION=5.4.6 PHP_VERSION=7.0
+  - LATEST=0 SHOPWARE_VERSION=5.3.7 PHP_VERSION=7.2
+  - LATEST=0 SHOPWARE_VERSION=5.3.7 PHP_VERSION=7.1
+  - LATEST=0 SHOPWARE_VERSION=5.3.7 PHP_VERSION=7.0
+  - LATEST=0 SHOPWARE_VERSION=5.2.27 PHP_VERSION=7.2
+  - LATEST=0 SHOPWARE_VERSION=5.2.27 PHP_VERSION=7.1
+  - LATEST=0 SHOPWARE_VERSION=5.2.27 PHP_VERSION=7.0
+
```

Comment:
should be one instead of two blank lines.

Reply:
That is intentional to seperate the environment from the test/script part